inductiva/_cli/cmd_tasks/kill.py

```diff
@@ -0,0 +1,41 @@
+"""Kills a tasks by id via CLI."""
+
+import inductiva
+
+
+def kill_task(args):
+    """Kills a task by id."""
+    if not args.yes:
+        prompt = input("Do you confirm you want to "
+                       f"kill {len(args.id)} tasks (y/[N])?")
+        confirm = prompt.lower() in ["y", "ye", "yes"]
+        if confirm:
+            print("Killing tasks.")
+        else:
+            print("Aborting the killing of tasks.")
+            return
```

Comment:
Remove most of it. The user already has all feedback from his immediate actions.
```suggestion
        if not confirm:
            print("Aborted.")
```